pilot/cmd/sidecar-injector/main.go

```diff
@@ -95,22 +111,63 @@ var (
 	}
 )
 
+func patchCert() error {
+	client, err := createClientset(flags.kubeConfigFile)
+	if err != nil {
+		return err
+	}
+	caCertPem, err := ioutil.ReadFile(flags.caCertFile)
+	if err != nil {
+		return err
+	}
+	err = util.PatchMutatingWebhookConfig(client.AdmissionregistrationV1beta1().MutatingWebhookConfigurations(),
```

Comment:
This could be rewritten to remove duplicate code (see below). 

```golang
for {
    err := util.PatchMutatingWebhookConfig(client.AdmissionregistrationV1beta1().MutatingWebhookConfigurations(), 
        sidecarWebhookConfigName, sidecarWebhookName, caCertPem)
    if err == {
        return nil
    }
	log.Errorf("Registering webhook %q:%q failed: %s. Retrying...", sidecarWebhookConfigName, sidecarWebhookName, err)
	time.Sleep(time.Second * 10)
}
```